charts/rekor/values.yaml

```diff
@@ -62,8 +62,8 @@ server:
     registry: gcr.io
     repository: projectsigstore/rekor-server
     pullPolicy: IfNotPresent
-    # crane digest gcr.io/projectsigstore/rekor-server:v0.12.0
-    version: "sha256:a72892d8d663624a89f7e7cb1113a0b38caf794a32d7e6ddc178f8e8aac2c8da"
+    # -- v0.12.0
```

Comment:
could we live the `crane digest` comment as a hint for anyone trying to update this chart later?

Reply:
yes, i did a request before to keep that

Reply:
That command is now documented in the CONTRIBUTING.md

Reply:
I have added the `crane digest` hints in all places now. Also in the places it wasn't there yet.